build/charts/antrea/README.md

```diff
@@ -44,6 +44,7 @@ Kubernetes: `>= 1.16.0-0`
 | antreaProxy.nodePortAddresses | list | `[]` | String array of values which specifies the host IPv4/IPv6 addresses for NodePort. By default, all host addresses are used. |
 | antreaProxy.proxyAll | bool | `false` | Proxy all Service traffic, for all Service types, regardless of where it comes from. |
 | antreaProxy.proxyLoadBalancerIPs | bool | `true` | When set to false, AntreaProxy no longer load-balances traffic destined to the External IPs of LoadBalancer Services. |
+| antreaProxy.serviceProxyName | string | `""` | The value of service.kubernetes.io/service-proxy-name label for AntreaProxy to match. If it is set, then AntreaProxy only handles the Service objects matching this label. The default value is empty string, which means that AntreaProxy will manage all Service objects without the mentioned label. |
```

Comment:
Guess we should revise the description. Consider:

`The value of the "service.kubernetes.io/service-proxy-name" label for AntreaProxy to match. If it is set, then AntreaProxy will only handle Services with the label that equals the provided value. If it is not set, then AntreaProxy will only handle Services without the "service.kubernetes.io/service-proxy-name" label, but ignore Services with the label no matter what is the value.

Reply:
Much better!